test/integration/targets/prepare_nxos_tests/tasks/main.yml

```diff
@@ -1,9 +1,55 @@
+---
 # NXAPI is enabled differently on NX-OS platforms.
 # Try both commands to enable NXAPI and ignore any errors.
 - name: enable nxapi on remote device
   nxos_config:
     lines:
-      - feature cli
```

Comment:
`feature cli` and `feature cli sandbox` were incorrectly used.  `nxapi` should be used instead.